README.md

```diff
@@ -1 +1,46 @@
-# portfolio-mobile
\ No newline at end of file
+# portfolio-mobile
+ This project is the third milestone of the portfolio mobile version
+
+
+## Built With
+
+- Major languages: HTML & CSS
+- A Linter for HTML & CSS
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Install : 
+A Linter for HTML & CSS
+
+
+
+
```

Comment:
- [ **OPTIONAL** ] I think it would be better to fulfill this section by adding some instructions on how to get your project locally like how to clone it. As I can see you wrote **`To get a local copy up and running follow these simple example steps.`** but those steps are not there. This will make your documentation looks more professional. I suggest you use this [template](https://github.com/tresorsawasawa/MyPortfolio#readme) to get some hints on how to fulfill it.